lib/services/notifications.js

```diff
@@ -12,9 +12,10 @@ const _ = require('lodash'),
  * @param {object} data
  * @returns {Promise}
  */
-function callWebhook(event, url, data) {
+function callWebhook(event, url, data, uri) {
   const headers = {
-      'X-Event': event
+      'X-Event': event,
+      'page-uri': uri
```

Comment:
Nice, I generally like this approach as it allows the JSON to remain untouched and allows webhooks to know the URI of the resource triggering the event.

I am wondering if we should use something like `Link: uri + ';rel=meta'`. The `Link` header has be proposed by IETF in 2010, although I do not know how widely it is used: https://www.rfc-editor.org/rfc/rfc5988.txt and https://www.w3.org/wiki/LinkHeader

One benefit of `Link` is that it is more generic and would cover the use-case of not only page publish events but also component instance publish events should we want to support them in the future.